subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/AbstractRealLifeAndroidBuildPerformanceTest.groovy

```diff
@@ -27,7 +27,7 @@ class AbstractRealLifeAndroidBuildPerformanceTest extends AbstractCrossVersionGr
 
     def setup() {
         runner.args = [AndroidGradlePluginVersions.OVERRIDE_VERSION_CHECK]
-        runner.targetVersions = ["6.7-20200723220251+0000"]
+        runner.targetVersions = ["6.7-20200824220048+0000"]
```

Comment:
There are a few rebaselined tests in this PR. Why have they been rebaselined? Can you revert the rebaselining on `master`?